test/jdk/java/lang/management/ThreadMXBean/VirtualThreadDeadlocks.java

```diff
@@ -98,6 +97,9 @@ public static void main(String[] args) throws Exception {
     private static void awaitBlocked(Thread thread) throws InterruptedException {
         while (thread.getState() != Thread.State.BLOCKED) {
```

Comment:
Thread.State.BLOCKED == thread.getState(). Does it make sense?

Reply:
Do you mean '==' or '!='? Both don't make sense to me, honestly. Could you please elaborate?